packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -291,7 +291,7 @@ export const VSelect = genericComponent<new <
           readonly
           placeholder={ placeholder }
           onClick:clear={ onClear }
-          onMousedown:control={ onMousedownControl }
+          onClick:control={ onClickControl }
```

Comment:
It should open on mousedown

Reply:
"Closing menu via click outside while open another menu via mousedown" is the cause of the issue, because "click outside" is always triggered after "mousedown":

Suppose there are two select components: Select 1 and Select 2

When menu of Select 1 is opening and user "mousedown" on Select 2

1. menu of Select 2 opens and it is placed on top of global stack
2. menu of Select 1 `globalTop` is false so click outside closeConditional is false
3. when user "mouseup" and trying to trigger "click outside" on menu of Select 1, false closeConditional stops it from firing

What was the initial purpose of opening menu via mousedown? In v2, it is open from click

Or unless we want to replace "click outside" with "mousedown outside" for this case

Reply:
@KaelWD [this comment](https://github.com/vuetifyjs/vuetify/issues/16819#issuecomment-1458231271) makes sense.

What would be the complexity level of adding `mousedown` to click-outside? Is it a reasonable assignment to getting familiar with click-outside internal?